middleware.go

```diff
@@ -48,7 +48,7 @@ func setRequestStartTime() Middleware {
 	}
 }
 
-//  Run the request through all middlewares
```

Comment:
You know, we could do these gofmt changes out of this PR since we still have wait on the centos7 change. I can post a quick PR.

Reply:
https://github.com/mozilla-services/autograph/pull/900

Reply:
I'll rebase my PR after that one lands to remove it - IIRC `lint-vet-fmt` job will fail without the linting changes. Not that it really matters, but I like keeping the CI green.

Reply:
Rebase has been completed!